lua/gluatest/runner/logger.lua

```diff
@@ -13,10 +13,18 @@ local ResultLogger = {}
 
 
 function ResultLogger.prefixLog( ... )
-    local prefixColor = SERVER and colors.server or colors.client
+    local prefixColor
+    local prefixText = "GLuaTest"
+    if SERVER then
+        prefixText = "sv_" .. prefixText
+        prefixColor = colors.server
+    else
+        prefixText = "cl_" .. prefixText
+        prefixColor = colors.client
+    end
```

Comment:
I don't _love_ this.

Could we maybe only prefix them if the clientside convar is enabled, or similar? Or, perhaps, only prefix the client tests?